public-ui/src/app/data-browser/charts/chart-biosex/chart-biosex-react.component.tsx

```diff
@@ -56,44 +101,61 @@ export class BioSexChartReactComponent extends React.Component<Props, State> {
 
   getSurveyChartOptions() {
     const {genderAnalysis: {analysisName, results}, selectedResult} = this.props;
-    baseOptions.chart.type = 'column';
-    baseOptions.plotOptions.column.groupPadding = 0.40;
-    baseOptions.plotOptions.series.pointWidth = 50;
-    baseOptions.legend.enabled = true;
-    baseOptions.yAxis.gridLineColor = '#ECF1F4';
-    baseOptions.title.style.color = '#262262';
-    baseOptions.title.style.fontSize = '22px';
-    baseOptions.color = '#2691D0';
-    baseOptions.xAxis.title.text = analysisName;
-    baseOptions.yAxis.title.text = 'Participant Count';
     const filteredResults = results.filter(
               r => r.stratum4 === selectedResult.stratum4);
     const {categories, series} = this.prepSurveyCategoriesAndData(filteredResults);
-    baseOptions.xAxis.categories = categories;
-    if ('dataOnlyLT20' in series[0]) {
-        baseOptions.yAxis.min = series[0].dataOnlyLT20 ? 20 : 0;
-        baseOptions.yAxis.labels = {
-            style: {
-                   fontSize: '14px',
-                   whiteSpace: 'wrap',
-                   textOverflow: 'ellipsis',
-                   color: '#262262'
-            },
-            formatter: function () {
-                   const label = this.axis.defaultLabelFormatter.call(this);
-                   // Change <= 20 count to display '<= 20'
-                   if (label <= 20) {
-                    return '&#8804; 20';
-                   }
-                   return label;
-                   },
-                   useHTML: true
-            };
-    }
-    baseOptions.series = series;
+    this.setCommonBioSexOptions(analysisName, categories, series);
     this.setState({options: baseOptions});
   }
 
+  prepEhrCategoriesAndData(genderAnalysisResults) {
```

Comment:
This function is almost identical to `prepSurveyCategoriesAndData`, looks like only `toolTipHelpText` is different. It would reduce duplicated code to combine them and have a condition in the tooltip content to display the correct text.

Reply:
Pulled the code into single method